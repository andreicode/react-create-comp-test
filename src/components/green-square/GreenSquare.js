import React from 'react';
import './style.scss';

const GreenSquare = (props) => {
  return (
    <div className="container">
      <p>GreenSquare1</p>
    </div>
  );
};

export default GreenSquare;
