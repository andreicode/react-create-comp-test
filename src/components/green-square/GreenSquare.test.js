import React from 'react';
import { render } from '@testing-library/react';
import GreenSquare from './GreenSquare';

describe('[COMPONENT] GreenSquare', () => {
  it('renders correctly', () => {
    const { container } = render(<GreenSquare />);
    expect(container).toMatchSnapshot();
  });
});
