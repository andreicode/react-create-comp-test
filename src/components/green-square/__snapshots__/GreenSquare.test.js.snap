// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[COMPONENT] GreenSquare renders correctly 1`] = `
<div>
  <div
    class="container"
  >
    <p>
      GreenSquare1
    </p>
  </div>
</div>
`;
